### DISCUSSION TITLE
Screen has no display but still responding to touch

### DISCUSSION THREAD

**Discussion by mattster98** [HASwitchPlate/HASPone#194] [Q&A]
I moved the unit into a new triple cover plate.  It was a little snug to get the display into the plate but nothing crazy.  After reassembling and powering up the system is up, on network, responding to touch, but nothing on the display other than backlight.

Wondering if there is any troubleshooting for what I may have broken or just need to replace the display.

**Comment by aderusha** [comment]
Your best bet will be to pull the thing open, remove the PCB from the mount to pull the board out of the rear enclosure, and then hook up to the USB port.  This will let you grab some debug logs that should tell you what is going on: https://github.com/HASwitchPlate/HASPone/wiki/Capture-HASPone-debug-output

If you can post the log results somewhere where we can take a look, we might be able to work out a solution together.

**Comment by mattster98** [comment]
Welp, I hooked it up again and it's working now.  No idea what was going on.  Thanks!
